pkg/knownguids/guids.go

```diff
@@ -10,1224 +10,1547 @@ package knownguids
-	*guid.MustParse("62E8F833-2B0A-4C19-A966-63C180588BE7"): "WinHost",
-	*guid.MustParse("BE0FEABA-3443-4919-9F3A-2D4216329EA9"): "WinNtAutoScan",
-	*guid.MustParse("F479E147-A125-11D4-BCFC-0080C73C8881"): "WinNtBlockIo",
-	*guid.MustParse("BD7E9A27-D6C5-416A-B245-5F507D95B2BD"): "WinNtBusDriverDxe",
-	*guid.MustParse("F0384FFD-8633-452F-9010-F6B7D2EAE2F1"): "WinNtFirmwareVolumePei",
-	*guid.MustParse("681F3771-6F1D-42DE-9AA2-F82BBCDBC5F9"): "WinNtFlashMapPei",
-	*guid.MustParse("29B3C4C6-E5AA-49E4-8CE0-2772F782DDC2"): "WinNtGopDxe",
-	*guid.MustParse("CA4233AD-847E-4E5D-AD3F-21CABFE5E23C"): "WinNtOemHookStatusCodeHandlerDxe",
-	*guid.MustParse("D233D6BD-F1B1-425A-BF45-5CAF2B88EDDC"): "WinNtOemHookStatusCodeHandlerPei",
-	*guid.MustParse("6B41B553-A649-11D4-BD02-0080C73C8881"): "WinNtSerialIoDxe",
-	*guid.MustParse("9C25E18B-76BA-43DA-A132-DBB0997CEFEF"): "WinNtSimpleFileSystemDxe",
-	*guid.MustParse("0C95A916-A006-11D4-BCFA-0080C73C8881"): "WinNtThunkDxe",
-	*guid.MustParse("D3AAD8DC-3A48-46AC-B1C7-28A9D3CF6755"): "WinNtThunkPPIToProtocolPei",
-	*guid.MustParse("2304DF80-E21D-4170-9C3C-113C878F7AC0"): "X86QemuLoadImageLib",
-	*guid.MustParse("49C481AF-1621-42F3-8FA1-27C64143E304"): "Xcode5SecPeiCpuExceptionHandlerLib",
-	*guid.MustParse("0EFC6282-F1E5-469A-8A70-194A8761F9AA"): "XenAcpiPlatformDxe",
-	*guid.MustParse("E3913319-96AC-4AC0-808B-8EDB8776A51C"): "XenArmGenericTimerVirtCounterLib",
-	*guid.MustParse("565EC8BA-A484-11E3-802B-B8AC6F7D65E6"): "XenBusDxe",
-	*guid.MustParse("401406DD-BCAC-4B91-9F4E-72A7FEBE4762"): "XenConsoleSerialPortLib",
-	*guid.MustParse("B5EE9A32-CA5A-49A8-82E3-ADA4CCB77C7C"): "XenHypercallLib",
-	*guid.MustParse("DE9BDC19-8434-47BB-BE3C-7F28F2101FD0"): "XenIoMmioLib",
-	*guid.MustParse("CF569F50-DE44-4F54-B4D7-F4AE25CDA599"): "XenIoPciDxe",
-	*guid.MustParse("7A567CC4-0E75-4D7A-A305-C3DB109B53AD"): "XenIoPvhDxe",
-	*guid.MustParse("6914C46F-D46E-48DC-9998-8A5F64F02553"): "XenPlatformHasAcpiDtDxe",
-	*guid.MustParse("DB54DBB7-8142-4EE5-9364-78C824B582EB"): "XenPlatformLib",
-	*guid.MustParse("F112A6EE-993A-4F0B-8295-E52029D9B4BA"): "XenPlatformPei",
-	*guid.MustParse("8C2487EA-9AF3-11E3-B966-B8AC6F7D65E6"): "XenPvBlkDxe",
-	*guid.MustParse("EC2557E8-7005-430B-9F6F-9BA109698248"): "XenRealTimeClockLib",
-	*guid.MustParse("52FE8196-F9DE-4D07-B22F-51F77A0E7C41"): "XenTimerDxe",
-	*guid.MustParse("40D1F8F5-4DFE-4E0F-9A15-B1DE9DC9F4ED"): "XenVirtMemInfoLib",
-	*guid.MustParse("338695EA-CA84-4FA2-9DA8-5C4BB87905C6"): "XenioFdtDxe",
-	*guid.MustParse("B7F50E91-A759-412C-ADE4-DCD03E7F7C28"): "XhciDxe",
-	*guid.MustParse("65E5746E-9C14-467D-B5B3-932A66D59F79"): "XhciPei",
-	*guid.MustParse("1831A379-2D48-45BD-9744-D4059D93815D"): "dp",
-	*guid.MustParse("0253F9FA-129A-4A8D-B12E-7DC2B6376302"): "dpDynamicCommand",
-	*guid.MustParse("8DC58D0D-67F5-4B97-9DFC-E442BB9A5648"): "tftp",
-	*guid.MustParse("A487A478-51EF-48AA-8794-7BEE2A0562F1"): "tftpDynamicCommand",
+var GUIDs = map[guid.GUID]string{}
```

Comment:
Update the comment at the top of this file about `THIS FILE IS GENERATED! DO NOT MODIFY!` and add instructions on how to update this list.